ironfish-cli/src/commands/service/bridge/deposit.ts

```diff
@@ -0,0 +1,176 @@
+import { Asset } from '@ironfish/rust-nodejs'
+import { CurrencyUtils, SendTransactionRequest, WebApi } from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import { IronfishCommand } from '../../../command'
+import { IronFlag, RemoteFlags } from '../../../flags'
+import { promptCurrency } from '../../../utils/currency'
+
+export class Deposit extends IronfishCommand {
+  static description = `Deposit coins to the bridge`
+
+  static flags = {
+    ...RemoteFlags,
+    endpoint: Flags.string({
+      char: 'e',
+      description: 'API host to sync to',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      env: 'IRONFISH_API_HOST',
+    }),
+    token: Flags.string({
+      char: 't',
+      description: 'API token to authenticate with',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      env: 'IRONFISH_API_TOKEN',
+    }),
+    account: Flags.string({
+      char: 'f',
+      description: 'The account to deposit from',
+    }),
+    to: Flags.string({
+      char: 't',
+      description: 'The public address of the bridge account',
+    }),
+    amount: IronFlag({
+      char: 'a',
+      description: 'Amount to deposit',
+      flagName: 'amount',
+    }),
+    fee: IronFlag({
+      char: 'o',
+      description: 'The fee amount in IRON',
+      minimum: 1n,
+      flagName: 'fee',
+    }),
+    confirm: Flags.boolean({
+      default: false,
+      description: 'Confirm without asking',
+    }),
+    expiration: Flags.integer({
+      char: 'x',
+      description:
+        'The block sequence after which the transaction will be removed from the mempool. Set to 0 for no expiration.',
+    }),
+    confirmations: Flags.integer({
+      char: 'c',
+      description:
+        'Minimum number of block confirmations needed to include a note. Set to 0 to include all blocks.',
+      required: false,
+    }),
+    assetId: Flags.string({
+      char: 'i',
+      description: 'The identifier for the asset to deposit',
+    }),
+    dest: Flags.string({
+      description: 'Eth public address to deposit to',
+      required: true,
+    }),
+  }
+
+  async start(): Promise<void> {
+    const { flags } = await this.parse(Deposit)
+
+    if (!flags.endpoint) {
+      this.log(
+        `No api host set. You must set IRONFISH_API_HOST env variable or pass --endpoint flag.`,
+      )
+      this.exit(1)
+    }
+
+    if (!flags.token) {
+      this.log(
+        `No api token set. You must set IRONFISH_API_TOKEN env variable or pass --token flag.`,
+      )
+      this.exit(1)
+    }
+
+    const api = new WebApi({ host: flags.endpoint, token: flags.token })
+
+    const client = await this.sdk.connectRpc()
+
+    const publicKey = (await client.wallet.getAccountPublicKey({ account: flags.account }))
+      .content.publicKey
+
+    const account =
+      flags.account ?? (await client.wallet.getDefaultAccount()).content.account?.name
+
+    const assetId = flags.assetId ?? Asset.nativeId().toString('hex')
```

Comment:
This is not a huge deal, but what do you think about leaving this just as the native asset for now? This could limit confusion before it's supported